class/50-host-classes

```diff
@@ -99,15 +75,6 @@ case $HOSTNAME in
     mysqlserver*)
         echo "FAIBASE KVMGUEST SYSADMIN MYSQLD"
         ;;
-    streamstation*)
```

Comment:
would like to keep streaming related ones. But please remove the FSF class from them all. and remove TRISQUEL